airflow/providers/amazon/aws/hooks/ecs.py

```diff
@@ -55,7 +55,7 @@ def should_retry_eni(exception: Exception):
     return False
 
 
-class EcsClusterStates(Enum):
+class EcsClusterStates(str, Enum):
```

Comment:
This for allow comparison `EcsClusterStates.ACTIVE == "ACTIVE"` unfortunetly class enum.StrEnum[¶](https://docs.python.org/3/library/enum.html#enum.StrEnum) available only in python 3.11

Reply:
Yeah, I was hoping to use StrEnum there but... not yet. :(  I didn't know you could do it this way, nice.